knative/Makefile

```diff
@@ -27,6 +27,10 @@ deploy: minio loadbalancer knative ## deploy knative
 killvms:
 	sudo pkill -SIGTERM firecracker
 
+# after cd ~/vhive && go install ./... you can run the deployer via:
+vhive-deployer:
+	~/go/bin/deployer -jsonFile ~/vhive/examples/deployer/functions.json -funcPath ~/vhive/configs/knative_workloads
```

Comment:
Maybe move to justfile?

```
$ nix shell .#vhive-examples -c deployer -jsonFile ~/vhive/examples/deployer/functions.json -funcPath ~/vhive/configs/knative_workloads
```

Reply:
moved to justfile. I will put vhive into a submodule in the next PR